### ISSUE TITLE
How we connect M95 with the aws iot using arduino 

### ISSUE THREAD

**Issue by PRADIPMAHAJAN** [vshymanskyy/TinyGSM#602]
Hey, I am trying to connect with m95 to the Arduino for communication between the client and AWS IoT, But have some issue I don't know how I upload my CA,CC and Private key with the command AT+QSECWRITE=\"RAM:cacert.pem\",1187,100 how to write my file to the serial port please help me. can we write complete file to the serial port how.

**Comment by SRGDamia1** [comment]
I'm sorry, I'm not familiar enough with the M95 or AWS certificates to help.

**Comment by OussamaNRCI** [comment]
Hi @PRADIPMAHAJAN Did you find a solution about load certificate to work with AWS IoT Core

**Comment by PRADIPMAHAJAN** [comment]
Yes, I got the solution buy a M95 Quectel GSM module and their is a document you can download from google m95 mqtt AT command PDF follow this pdf store certificates, private key or Root cert in NVRAM of M95 and try to connect.
